examples/subplots_axes_and_figures/secondary_axis.py

```diff
@@ -79,6 +79,17 @@ def one_over(x):
 # the data, and is derived empirically.  In that case we can set the
 # forward and inverse transforms functions to be linear interpolations from the
 # one data set to the other.
+#
+# .. note ::
```

Comment:
Apparently this works. However, I'd rather stay exactly with the [documented syntax](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=..%20note#directive-note), because Sphinx/ReST can be quite picky and I don't want to risk that this will break with a future release of Sphinx.

```suggestion
# .. note::
```

Reply:
I actually copied that syntax from the example right above it. I'll go ahead and fix both.